src/main/java/net/mcreator/workspace/WorkspaceFolderManager.java

```diff
@@ -105,6 +105,14 @@ public List<String> getStructureList() {
 		return structures;
 	}
 
+	public File getVillagerProfessionTextureFileForName(String professionTextureName) {

```

Comment:
Right now, all other texture types belong to other, not to a subfolder. So show preview directly from other folder, as other mod elements do

Reply:
Done!